runtime/doc/builtin.txt

```diff
@@ -4327,6 +4329,27 @@ getregion({pos1}, {pos2} [, {opts}])			*getregion()*
 		Can also be used as a |method|: >
 			getpos('.')->getregion(getpos("'a"))
 <
+getregionpos({pos1}, {pos2} [, {opts}])            *getregionpos()*
+		Returns the list of position from {pos1} to {pos2} from a
+		buffer.
+		See |getregion()| for the arguments and notes.
+		Note: When the selection is blockwise, it returns a pair of
+		positions for every line.
+
+		The position is a |List| with four numbers:
+		    [bufnum, lnum, col, off]
+		"bufnum" is zero, unless a mark like '0 or 'A is used, then it
+		is the buffer number of the mark.
```

Comment:
Is "bufnum is zero" referring to the input positions or the output? Can it support non-current buffer?

Reply:
I have changed it.